test_pull_requests

```diff
@@ -53,7 +55,9 @@ opts = GetoptLong.new(
 ["--help",               "-h",      GetoptLong::NO_ARGUMENT],
 ["--merge_pull_request",            GetoptLong::REQUIRED_ARGUMENT],
 ["--local_merge_pull_request",      GetoptLong::REQUIRED_ARGUMENT],
+["--base_commit",                   GetoptLong::REQUIRED_ARGUMENT],
 ["--test_merge_pull_request",       GetoptLong::REQUIRED_ARGUMENT],
+["--can_skip_merge_tests",          GetoptLong::REQUIRED_ARGUMENT],
```

Comment:
Yep, just there for testing

On Aug 3, 2016 7:31 PM, "Steve Kuznetsov" <notifications@github.com> wrote:

> In test_pull_requests
> <https://github.com/openshift/test-pull-requests/pull/31#discussion_r73440149>
> :
>
> >  ["--test_merge_pull_request",       GetoptLong::REQUIRED_ARGUMENT],
> > +["--can_skip_merge_tests",          GetoptLong::REQUIRED_ARGUMENT],
>
> Should the presence of the config in the .test_pull_requests.json be
> enough to indicate that we want this behavior?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/test-pull-requests/pull/31/files/0ab802fdc4cc9aa5603976806dd8661979ce0fb1#r73440149>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABd0FGEGE-VWpVNJ8k0OvEY6pSufbnkBks5qcSSzgaJpZM4JcBHJ>
> .
>